Frontend/src/Components/Motor_Motion/Motor_Motion.js

```diff
@@ -0,0 +1,448 @@
+                        <Text fontSize='xs'>Speed</Text>
+                        <Spacer/>
+                        <Text fontSize='xs'>{props.data?.speed[0].toFixed(0) ?? -1.0} {CONSTANTS.speed.UNIT}</Text>
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={4}
+                    colStart={3}
+                    borderTop='1px'
+                    borderLeft='1px'
+                    borderRight='1px'
+                    pl='1' 
+                    pr='1'
+                    pb='1'
+                >
+                    <RangeCell
+                        fontSize='xs'
+                        rangeBarH='1vh'
+                        label='Motor Power'
+                        data={props.data?.motor_power[0] ?? -1.0}
+                        digits={0}
+                        unit={CONSTANTS.motor_power.UNIT}
+                        min={CONSTANTS.motor_power.MIN}
+                        max={CONSTANTS.motor_power.MAX}
+                    />
+                </GridItem>
+                <GridItem
+                    rowStart={5}
+                    colStart={3}
+                    borderTop='1px'
+                    borderLeft='1px'
+                    borderRight='1px'
+                    borderBottom='1px'
+                    pl='1' 
+                    pr='1'
+                    pb='1'
+                >
+                    <RangeCell
+                        fontSize='xs'
+                        rangeBarH='1vh'
+                        label='Motor Current'
+                        data={props.data?.motor_current[0] ?? -1.0}
+                        digits={0}
+                        unit={CONSTANTS.motor_current.UNIT}
+                        min={CONSTANTS.motor_current.MIN}
+                        max={CONSTANTS.motor_current.MAX}
+                    />
+                </GridItem>
+                <GridItem
+                    rowStart={2}
+                    colStart={4}
+                    pl='1'
+                    pr='1'
+                >
+                    <Center
+                        borderColor={borderCol}
+                        borderTop='1px'
+                        borderLeft='1px'
+                        borderRight='1px'
+                        h='100%'
+                    >
+                        <Text fontSize='sm'>Setpoints</Text>
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={3}
+                    colStart={4}
+                    pl='1'
+                    pr='1'
+                >
+                    <Center
+                        bg={props.data?.crz_spd_setpt[0] ? greenBgCol : redBgCol}
+                        borderColor={borderCol}
+                        borderTop='1px'
+                        borderLeft='1px'
+                        borderRight='1px'
+                        h='100%'
+                    >
```

Comment:
```suggestion
                        borderColor={borderCol}
                        borderTopWidth='1px'
                        borderLeftWidth='1px'
                        borderRightWidth='1px'
                        h='100%'
                    >
```
I'm also not sure if the `h='100%'` is needed, but it's not hurting anything